src/bin/commands/trimmer.rs

```diff
@@ -109,6 +109,11 @@ fn create_writer<P: AsRef<Path>>(name: P) -> Result<BufWriter<File>, Error> {
     Ok(BufWriter::new(File::create(name)?))
 }
 
+fn check_extension(p: &Path) -> bool {
+    let ext = p.extension().map_or("", |v| v.to_str().unwrap_or(""));
+    ["bgz", "gz"].contains(&ext)
```

Comment:
Cna we use: https://github.com/fulcrumgenomics/fgoxide/blob/e3a3068c451d8a06491e499075032e2075c41e7c/src/io/mod.rs#L56

Or better yet:

https://github.com/fulcrumgenomics/fgoxide/blob/e3a3068c451d8a06491e499075032e2075c41e7c/src/io/mod.rs#L82C8-L82C20

Reply:
see: https://github.com/fulcrumgenomics/fgoxide/pull/11